ci/jobs/pulp-dev.yaml

```diff
@@ -81,22 +81,17 @@
             steps:
                 - sonar:
                     sonar-name: 'Sonar Test Server'
-        - conditional-step:
-            condition-kind: current-status
-            condition-worst: FAILURE
-            condition-best: UNSTABLE
```

Comment:
Aha. So formerly, we'd only capture logs if the build result was one of UNSTABLE or FAILURE. But that leaves out the other states of SUCCESS, NOT_BUILD and ABORTED, and there's value in capturing logs in those other states too. Right?

Reference: http://docs.openstack.org/infra/jenkins-job-builder/builders.html#builders.conditional-step